### PR TITLE
Exiled outdated will give more information

### PR DESCRIPTION
**Pull request by louis1706**

### CODE REVIEW

File: Exiled.Loader/LoaderPlugin.cs [hunk 1]
```diff
@@ -52,7 +52,7 @@ public void Enable()
                     GameCore.Version.BackwardCompatibility,
                     GameCore.Version.BackwardRevision))
             {
-                ServerConsole.AddLog("Exiled is outdated, please update to the latest version. Wait for release if still shows after update.", ConsoleColor.DarkRed);
+                ServerConsole.AddLog($"Exiled is outdated, please update to the latest version. Wait for release if still shows after update.\nSCP:SL = {GameCore.Version.VersionString} Exiled Supported Version = {AutoUpdateFiles.RequiredSCPSLVersion}", ConsoleColor.DarkRed);
```
Comment (Misaka-ZeroTwo) [inline Exiled.Loader/LoaderPlugin.cs:55]:
```suggestion
                ServerConsole.AddLog($"Exiled is outdated, please update to the latest version. Wait for release if still shows after update.\nSCP:SL: {GameCore.Version.VersionString} Exiled Supported Version: {AutoUpdateFiles.RequiredSCPSLVersion}", ConsoleColor.DarkRed);
```

**Review by Misaka-ZeroTwo — APPROVED** [review]

**Comment by Misaka-ZeroTwo** [comment]
Can you move 
``Log.Info($"Loading EXILED Version: {Assembly.GetExecutingAssembly().GetCustomAttribute<AssemblyInformationalVersionAttribute>().InformationalVersion}");``
before that game version check?

**Comment by louis1706** [comment]
> Can you move `Log.Info($"Loading EXILED Version: {Assembly.GetExecutingAssembly().GetCustomAttribute<AssemblyInformationalVersionAttribute>().InformationalVersion}");` before that game version check?

Why ?

**Comment by Misaka-ZeroTwo** [comment]
> > Can you move `Log.Info($"Loading EXILED Version: {Assembly.GetExecutingAssembly().GetCustomAttribute<AssemblyInformationalVersionAttribute>().InformationalVersion}");` before that game version check?
> 
> Why ?

Because it's not printing current exiled version if the game version check failed.

**Comment by louis1706** [comment]
> > > Can you move `Log.Info($"Loading EXILED Version: {Assembly.GetExecutingAssembly().GetCustomAttribute<AssemblyInformationalVersionAttribute>().InformationalVersion}");` before that game version check?
> > 
> > 
> > Why ?
> 
> Because it's not printing current exiled version if the game version check failed.

read the code again if not comptible it's will send the version

**Comment by Misaka-ZeroTwo** [comment]
> > > > Can you move `Log.Info($"Loading EXILED Version: {Assembly.GetExecutingAssembly().GetCustomAttribute<AssemblyInformationalVersionAttribute>().InformationalVersion}");` before that game version check?
> > > 
> > > 
> > > Why ?
> > 
> > 
> > Because it's not printing current exiled version if the game version check failed.
> 
> read the code again if not comptible it's will send the version

It doesn't send EXILED VERSION.

**Comment by louis1706** [comment]
> > > > > Can you move `Log.Info($"Loading EXILED Version: {Assembly.GetExecutingAssembly().GetCustomAttribute<AssemblyInformationalVersionAttribute>().InformationalVersion}");` before that game version check?
> > > > 
> > > > 
> > > > Why ?
> > > 
> > > 
> > > Because it's not printing current exiled version if the game version check failed.
> > 
> > read the code again if not comptible it's will send the version
> 
> It doesn't send EXILED VERSION.

Oh okay yes got it

**Review by IRacle1 — APPROVED** [review]

**Pull request merged**